modules/oe_translation_epoetry/src/ContentFormatter/HtmlFormatter.php

```diff
@@ -91,6 +91,16 @@ public function import(string $file, TranslationRequestInterface $request): arra
       return [];
     }
 
+    $assets = $xml->xpath("//div[@class='asset']");
+    if (!$assets) {
+      return [];
+    }
+
+    $asset_id = (string) $assets[0]->attributes()['id'];
+    if ($asset_id !== 'item-' . $request->id()) {
+      throw new \Exception('The translation request file does not match the translation request.');
```

Comment:
Is it ok to use the global \Exception case in this case? It would suggest that all other exceptions (like type errors or other internal stuff) are considered as `The ePoetry notification did not provide a valid translation` by the logger. This may be misleading while investigating issues.

Reply:
Well, this exception is thrown only if the request IDs dont match, not for any other reasons, so the message is returned only in this case.